postgis/postgis.sql.in

```diff
@@ -2881,7 +2900,8 @@ BEGIN
 			RAISE DEBUG '%', sql;
 			EXECUTE sql;
 		ELSIF (rec.default_version = rec.installed_version AND rec.installed_version ILIKE '%dev') OR
-			(@extschema@._postgis_pgsql_version() != @extschema@._postgis_scripts_pgsql_version())
+			--(@extschema@._postgis_pgsql_version() != @extschema@._postgis_scripts_pgsql_version())
+			(public._postgis_pgsql_version() != public._postgis_scripts_pgsql_version())
```

Comment:
Why the hard-coded `public` here instead of keeping @extschema@ ?

Reply:
Good catch. Testing purposes during dev.